app/pods/publications/template.hbs

```diff
@@ -29,32 +29,34 @@
               />
               {{#if showSearchResults}}
                 <div class="autocomplete-results auk-search-results-list">
-                  {{#each this.searchResults as |result|}}
-                    {{#if result.publicationFlowId}}
-                      <WebComponents::AuDropdown::Item
-                              @class="auk-search-results-list__item">
-                        <LinkTo
-                                data-test-publication-search-result
-                                @route="publications.publication.documents"
-                                @model={{result.publicationFlowId}}
-                        >
-                          {{result.subcaseSubTitle}}
-                          <br>
-                          <span class="auk-u-muted auk-u-text-small">{{result.publicationFlowNumber}}</span>
-                        </LinkTo>
-                      </WebComponents::AuDropdown::Item>
-                    {{else}}
-                      <WebComponents::AuDropdown::Item
-                              {{on "click" (fn this.startPublicationFromCaseId result.id)}}
-                              @class="auk-search-results-list__item">
-                        {{result.subcaseSubTitle}}
-                        <br>
-                        <span class="auk-u-muted auk-u-text-small">
-                          {{t "publication-not-started-yet"}}
-                        </span>
-                      </WebComponents::AuDropdown::Item>
-                    {{/if}}
-               {{/each }}
+                  {{#if (gt this.searchResults.length 0)}}
+                    {{#each this.searchResults as |result|}}
+                      {{#if result.publicationFlowId}}
```

Comment:
Deze structuur, je zou denken dat je alleen wil zoeken op dossiers met publications, dus zou je altijd een publicationFlowId moeten hebben ?

Reply:
binnekort zelf meerdere, als 1 dossier meerdere publicaties gaat kunnen hebben (probleem dat we dan moeten oplossen)

Reply:
Klopt, zit ook in de query

Reply:
aangepast